cpp/oneapi/dal/backend/primitives/selection/kselect_by_rows_simd.hpp

```diff
@@ -177,6 +172,7 @@ class kselect_by_rows_simd : public kselect_by_rows_base<Float> {
         });
         return event;
     }
+    static constexpr std::int64_t preffered_wg_size = 128;
```

Comment:
Why you can't use https://github.com/oneapi-src/oneDAL/blob/2bc78f2d2502fdd6dc3a547ab91f36cb3bccbbc8/cpp/oneapi/dal/backend/common.hpp#L199
instead?

Reply:
Can you please not silently resolve without changes but give an explanation?